exit status: 1

stdout = """
def foo: Bool -> #error = fun(_: Bool) => #error;
types of expressions:
span   | source                                   | type
-------|------------------------------------------|-----------------------------
10..34 | fun(false: Bool) => true                 | Bool -> #error
21..25 | Bool                                     | Type
30..34 | true                                     | Bool
types of patterns:
span   | source                                   | type
-------|------------------------------------------|-----------------------------
14..19 | false                                    | Bool
"""

stderr = """
error: inexhaustive pattern match
  ┌─ tests/elab/errors/pattern_matching/inexhaustive_fun_lit.pion:1:15
  │
1 │ def foo = fun(false: Bool) => true;
  │               ^^^^^

aborting due to 1 previous error
"""
